### PR TITLE
Add total balance and chain breakdown to events

### PR DESCRIPTION
**Pull request by everdimension**

### CODE REVIEW

File: src/ui/shared/requests/PortfolioValue/getAddressPortfolio.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { addressPortfolio, client, type Portfolio } from 'defi-sdk';
+import { addressPortfolioDecomposition, client } from 'defi-sdk';
+import { type PortfolioDecomposition } from 'defi-sdk';
 
 export function getAddressPortfolio({
```
Comment (zerts) [inline src/ui/shared/requests/PortfolioValue/getAddressPortfolio.ts:4]: `getAddressPortfolioDecomposition` ?

File: src/shared/analytics/shared/getTotalWalletsBalance.ts [hunk 2]
```diff
@@ -1,25 +1,23 @@
-import type { Portfolio } from 'defi-sdk';
+import type { PortfolioDecomposition } from 'defi-sdk';
 import { client } from 'defi-sdk';
 
 export async function getAddressesPortfolio(addresses: string[]) {
```
Comment (zerts) [inline src/shared/analytics/shared/getTotalWalletsBalance.ts:4]: `getAddressesPortfolioDecomposition`?

File: src/shared/analytics/shared/mixpanel-data-helpers.ts [hunk 3]
```diff
@@ -157,7 +167,8 @@ export async function getBaseMixpanelParams(account: Account) {
     num_zerion_wallets: ownedAddressesCount,
     num_connected_wallets: 0,
     num_wallet_groups: groups?.length ?? 0,
-    total_balance: portfolioStats?.totalValue ?? 0,
+    total_balance: portfolioStats?.portfolio?.total_value ?? 0,
+    ...getChainBreakdown(portfolioStats?.portfolio ?? null),
```
Comment (zerts) [inline src/shared/analytics/shared/mixpanel-data-helpers.ts:171]: Do we need this breakdown in user_properties?

**Review by zerts — APPROVED** [review]

File: src/shared/analytics/shared/mixpanel-data-helpers.ts (hunk 3 above)
Comment (everdimension) [reply src/shared/analytics/shared/mixpanel-data-helpers.ts:171]: Unfortunately :)

**Pull request merged**